src/jswrap_flash.c

```diff
@@ -114,8 +116,10 @@ doesn't return all `0xFF`) you'll need to call `erasePage` to clear the
 entire page.
  */
 void jswrap_flash_write(JsVar *data, int addr) {
-  if (jsvIsUndefined(data))
-    return jsExceptionHere(JSET_ERROR, "Data is not defined");
+  if (jsvIsUndefined(data)) {
+    jsExceptionHere(JSET_ERROR, "Data is not defined");
```

Comment:
Does this really matter, or is it a personal code style thing? I've been doing it a lot as it seems a bit more concise.